docs/setting-up-cloudgate-aws.md

```diff
@@ -0,0 +1,62 @@
+# Setting up CloudGate with AWS
+
+Cloudgate works using the STS functionality of AWS IAM.
+There are 3 main paths inside the AWS code:
+1. Get the list of accounts/roles the user can assum
+2. Get an STS token for an account+Role.
+3. Get a console URL redirection for an Account+Role.
+
+Cloudgate uses an IAM user that is allowed to list roles in each account and is also trusted to assume certain roles in that account. Cloudgate assumes there is a common role in each account that permits is to list the roles in that account. By default the name of this role is: `CPEBrokerRole`, but it can be selected on the configuration (as aws_list_roles_role_name). Whatever name it is it MUST be the same across all accounts.
```

Comment:
It should also be able to use an IAM to list roles, correct?
/s/is to list/it to list/
s/on the configuration/in the configuration/